Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/.gitignore	(revision d7cb849f3f9ac70b61b5bd28a79005578819f37f)
+++ .idea/.gitignore	(date 1605912128360)
@@ -1,0 +1,73 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+env/
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+*.egg-info/
+.installed.cfg
+*.egg
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*,cover
+.hypothesis/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
+*.log
+
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+target/
+
+#Ipython Notebook
+.ipynb_checkpoints
+.idea/encodings.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/pythonic-demos.iml
+.idea/vcs.xml
+.DS_Store
+.env
+.idea/workspace.xml
+.idea/dataSources/35b5cf00-25b0-486c-ad0e-a4924f786a9c.xml
+.idea/dataSources/8bb4bd1d-6254-49eb-8c86-6e28d7023c1b.xml
+.idea/inspectionProfiles/Project_Default.xml
\ No newline at end of file
Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># This is a sample Python script.\r\n######## BRAND NEW new_branch #################\r\n################ main.py #################\r\n# THIS LINE IS ADDED in new_branch to main.py\r\n# 1) THIS LINE IS ADDED AFTER new_branch IS PUSHED\r\nfrom exception import *\r\n\r\n\r\n# Press Shift+F10 to execute it or replace it with your code.\r\n# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.\r\n\r\n\r\ndef print_hi(name):\r\n    # Use a breakpoint in the code line below to debug your script.\r\n    print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.\r\n\r\n\r\n# Press the green button in the gutter to run the script.\r\nif __name__ == '__main__':\r\n    print_hi('In  main()....')\r\n    myException()\r\n\r\n# See PyCharm help at https://www.jetbrains.com/help/pycharm/
===================================================================
--- main.py	(revision d7cb849f3f9ac70b61b5bd28a79005578819f37f)
+++ code/main.py	(date 1605979754848)
@@ -20,4 +20,5 @@
     print_hi('In  main()....')
     myException()
 
-# See PyCharm help at https://www.jetbrains.com/help/pycharm/
\ No newline at end of file
+# See PyCharm help at https://www.jetbrains.com/help/pycharm/
+
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"Python 3.9 (MyException)\" project-jdk-type=\"Python SDK\" />\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/misc.xml	(revision d7cb849f3f9ac70b61b5bd28a79005578819f37f)
+++ .idea/misc.xml	(date 1605980696651)
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (MyException)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9" project-jdk-type="Python SDK" />
 </project>
\ No newline at end of file
Index: .idea/dictionaries/admin.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/dictionaries/admin.xml	(date 1605979756331)
+++ .idea/dictionaries/admin.xml	(date 1605979756331)
@@ -0,0 +1,3 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="admin" />
+</component>
\ No newline at end of file
Index: .idea/MyException.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"PYTHON_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <excludeFolder url=\"file://$MODULE_DIR$/venv\" />\r\n    </content>\r\n    <orderEntry type=\"inheritedJdk\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n  <component name=\"PyDocumentationSettings\">\r\n    <option name=\"format\" value=\"PLAIN\" />\r\n    <option name=\"myDocStringFormat\" value=\"Plain\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/MyException.iml	(revision d7cb849f3f9ac70b61b5bd28a79005578819f37f)
+++ .idea/MyException.iml	(date 1605980696536)
@@ -4,7 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 3.9" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
   <component name="PyDocumentationSettings">
diff --git exception.py code/exception.py
rename from exception.py
rename to code/exception.py
